.tools/readmes/writeme.py

```diff
@@ -147,18 +147,21 @@ def main():
                 logging.exception("Exception rendering %s", id)
                 failed.append(id)
 
-    done_list = "\n\t".join(sorted(written))
-    skip_list = "\n\t".join(sorted(skipped))
-    non_writeme_list = "\n\t".join(sorted(non_writeme))
-    unchanged_list = "\n\t".join(sorted(unchanged))
-    logging.debug(f"Skipped:\n\t{skip_list}")
-    logging.info(f"Wrote:\n\t{done_list}\nUnchanged:\n\t{unchanged_list}")
-    logging.warning(f"Non-WRITEME READMES:\n\t{non_writeme_list}")
-    if len(failed) > 0:
+    done_list = "\n".join(f"Wrote {f}" for f in sorted(written))
+    skip_list = "\n".join(f"Skipped {f}" for f in sorted(skipped))
+    print(skip_list or "(None Skipped)")
+    print(done_list or "(None Written)")
+    if unchanged:
+        unchanged_list = "\n".join(f"Unchanged {f}" for f in sorted(unchanged))
+        print(unchanged_list)
+    if non_writeme:
+        non_writeme_list = "\n\t".join(sorted(non_writeme))
+        print(f"Non-WRITEME READMES:\n\t{non_writeme_list}")
+    if failed:
         failed_list = "\n\t".join(failed)
-        logging.error(f"READMEs with incorrect formatting:\n\t{failed_list}")
-        logging.error("Rerun writeme.py to update README links and sections.")
-    logging.info("Run complete.")
+        print(f"READMEs with incorrect formatting:\n\t{failed_list}")
+        print("Rerun writeme.py to update README links and sections.")
+    print("WRITEME Run completed.")
```

Comment:
In cases like these, shouldn't we be taking opportunities to implement logging instead of print statements for tooling, especially when we already have it configured?

Reply:
Do you like the logging prefixe with `INFO:root:` ? I don't, so that's why I nixed it here. And these aren't "levels" of severity, either they get written to stdout or not.

This is a user-focused development tool. We talked a bit about this in your PR that added the error logging (which I should also change, if I do this. Done.). We're still trying to strike the balance of machine and human friendliness for this tool. I think this is an improvement, but if you disagree that this formatting is better, I'll split them into two.

Reply:
Wait I take that back there `skip` is not worth printing every time...

Reply:
Fair points.